SConstruct

```diff
@@ -217,8 +217,10 @@ config_options = [
         }),
     PathOption(
         "prefix",
-        """Set this to the directory where Cantera should be installed. On Windows
-           systems, '$ProgramFiles' typically refers to "C:\Program Files".""",
+        """Set this to the directory where Cantera should be installed. If the Python
+           executable found during compilation is managed by 'conda', the
+           installation 'prefix' will default to the corresponding environment. On
+           Windows systems, '$ProgramFiles' typically refers to "C:\Program Files".""",
```

Comment:
I think this should also mention the other preconditions for chosing the conda environment as the `prefix`.